### PR TITLE
Supergloo trafficshifting

### PR DESCRIPTION
**Pull request by ilackarms**
tutorial + concepts for traffic shifting with istio

### CODE REVIEW

File: supergloo/docs/mesh/install.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 ---
 title: "Installing a Mesh"
+<<<<<<< HEAD
```
Comment (christian-posta) [inline supergloo/docs/mesh/install.md]: Looks like a merge made it in?

**Comment by christian-posta** [comment]
Hey, can you add a `clean` target to the Makefile too? Just like the other projects?

**Comment by ilackarms** [comment]
i think the `clean` target only exists in the root `Makefile`

File: supergloo/docs/mesh/_index.md [hunk 2]
```diff
@@ -11,4 +11,6 @@ Note that this documentation is still under heavy development.
 Mesh Management Tutorials:
 
 - [Installing Istio with SuperGloo](install)
-- AWS AppMesh (coming soon)
\ No newline at end of file
+- [Installing the Bookinfo Example](bookinfo)
```
Comment (rickducott) [inline supergloo/docs/mesh/_index.md]: nit: this is called "Deploying the Bookinfo Example"

**Pull request merged**